web/src/app/projects/view/view.component.ts

```diff
@@ -91,6 +94,31 @@ export class ViewProjectComponent implements OnInit, OnDestroy {
       .subscribe();
   }
 
+  /**
+   * Increase the followers count by 1 and update the followings in the users collection
+   */
+  incrementFollowers(): void {
+    this.projectService.updateFollowers(this.project.uid, 1).subscribe();
+    this.userService.updateFollowings(this.authService.profile.uid, this.project.uid, true).
+      pipe(
+        tap(() => this.setFollower())
+    )
+    .subscribe();
```

Comment:
Empty `subscribe` does not seem quote right, can the `setFollower` be in the `subscribe`?